### DISCUSSION TITLE
Cursor is hidden in the indented line but I would like it to be showing anywhere.

### DISCUSSION THREAD

**Discussion by hrcHarry** [lukas-reineke/indent-blankline.nvim#607] [Q&A]
[Edited]

Hi, the auxiliary line to show the indentation is very helpful. But I have an issue.

I run neovim on the default terminal of Ubuntu 18.04 which runs on VirtualBox 7.0.

In addition, the color scheme is Nordfox of [nightfox.nvim](https://github.com/EdenEast/nightfox.nvim).

I configure [indent-blankline.nvim](https://github.com/lukas-reineke/indent-blankline.nvim) in the simplest way:
```
require("indent_blankline").setup {}
```
And the cursor always hides itself when it is at the no-content region of the indented line.
![](https://upload.cc/i1/2023/08/21/iZ1NUl.gif)

How to make it appear anyway? I would like the cursor to be showing at any place.

Thanks.

**Comment by lukas-reineke** [comment]
I've never seen that before. What terminal are you using?

**Reply by hrcHarry** [reply]
Hi, thanks for replying. 

I run neovim on the default terminal of Ubuntu 18.04 which runs on VirtualBox 7.0.

In addition, the color scheme is Nordfox of [nightfox.nvim](https://github.com/EdenEast/nightfox.nvim).

**Comment by lukas-reineke** [comment]
Please open a normal issue if this is still happening in version 3

**Comment by caligian** [comment]
I am experiencing the same issue here. Using xfce4-terminal. However the same does not happen in nvim-qt
